src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h

```diff
@@ -55,14 +55,14 @@ typedef struct EnBigpo {
 
 // well ver is regular, dampe basement ver is summoned
 // on spawn, 3/possible fires are turned into chosenfire
```

Comment:
```suggestion
// well version is "regular" (spawns automatically), dampe basement version is "summoned"
// on scene enter, 3 "possiblefire" are turned into "chosenfire" at random
```
Not the scope of the PR but this documentation should be more specific